### PR TITLE
My first Pull request

### PR DESCRIPTION
**Pull request by PushKK**
6 small fix:
1. missing letters in comments - 2 commits,
2. images 22x22 in folder 24x24,
3. sorting countries and add new country in holidays.xml,
4. add instruction for install GRAMPS in fresh Linux (Ubuntu),
5. import/open dialog in RU.po.

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -3153,21 +3153,21 @@ msgid ""
...
 "\n"
-"Обновление это сложная задача. которая может привести к порче Вашего "
-"семейного древа, если оно будет прервано или окажется неудачным.\n"
+"Обновление - это сложная задача, которая может привести к порче Вашего "
+"Семейного древа, если будет прервано или окажется неудачным.\n"
 "\n"
-"Если Вы ещё не сделали резервной копии Вашего семейного древа, то запустите "
-"%(bold_start)sстарую%(bold_end)s версию Gramps\n"
+"Если Вы ещё не сделали резервной копии Вашего Семейного древа, то запустите "
+"%(bold_start)sстарую%(bold_end)s версию Gramps"
```
Comment (vantu5z) [inline po/ru.po:3170]: you should add space after "Gramps"

**Comment by vantu5z** [comment]
@PushKK, thanks for Russian updates.

File: po/ru.po [hunk 2]
```diff
@@ -30389,7 +30386,6 @@ msgid "Field '%(fldname)s' not found"
 msgstr "Поле '%(fldname)s' не найдено"
 
 #: ../gramps/plugins/lib/libprogen.py:491
-#, fuzzy, python-format
```
Comment (vantu5z) [inline po/ru.po:30392]: Нужно удалить только `fuzzy`, а `python-format` оставить

**Comment by prculley** [comment]
I have rebased and merged this manually.  I did combine a few of the ru.po file commits into a single commit.

In the future, it would be best to keep to one topic per PR; these were in a lot of different subject areas.  Although you correctly kept it to one topic per commit.  This will make it easier for us to review, and will avoid the possibility that we would reject your PR because we were not ready to accept a single part of your PR.

**Pull request closed without merging**